### PR TITLE
README

### PR DESCRIPTION
**Pull request by corinnekrych**
@edewit @danielpassos 
Could you review pls, you should also look at Windows README at the same time to ensure we're consistent in the README instructions

### CODE REVIEW

**Comment by edewit** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,3 +1,96 @@
...
+
+The Cloud App proxies the login client call to a SAML Service (e.g. fetching a URL to display in the WebView for IdP login) through the use of a SAML service (See Configuration section).
+
+The client app demoes the usage of the two SAML Service endpoints:
+- `sso/session/login_host` to get the SAML login page URL. A WebView is used to display the SAML login page.
+- `sso/session/valid` to get the user information once the login was successful. 
+
+Before running the project you will need some configuration.
+
+## Configuration
```
Comment (danielpassos) [inline README.md:15]: @corinnekrych @edewit This section is not about configuration for client app, IMHO it need to be moved to the service, cloud or client template. Wdyt?

**Comment by abstractj** [comment]
@corinnekrych @danielpassos @edewit let's move this document to https://github.com/fheng/fh-doxy/tree/FH-1629, please.

**Comment by corinnekrych** [comment]
@abstractj @abstractj no problem with me, once this is moved to the proper section in doxy we can always make a link to it as a pre-requisites.

**Comment by abstractj** [comment]
@corinnekrych thank you!

**Comment by danielpassos** [comment]
It means, let's close it?

**Comment by abstractj** [comment]
@corinnekrych I'm closing this PR, because the content was merged here https://github.com/fheng/fh-doxy/commits/FH-1629

**Pull request closed without merging**